apps/api/src/server/app/case-officer/case-officer.confirm.test.js

```diff
@@ -6,9 +6,19 @@ import { app } from '../../app.js';
 const request = supertest(app);
 
 test('should submit confirmation of the outcome of LPA questionnaire', async (t) => {
-	const resp = await request.post('/case-officer/id:/confirm')
+	const resp = await request.post('/case-officer/6/confirm')
+		.send({});
+	t.is(resp.status, 200);
+});
+
+test('should submit confirmation of an incomplete outcome of LPA questionnaire', async (t) => {
+	const resp = await request.post('/case-officer/7/confirm')
 		.send({
-			Complete: true
+			reason:{
+				applicationPlansToReachDecisionMissingOrIncorrect: true,
+				applicationPlansToReachDecisionMissingOrIncorrectDescription: 'Some description'
+			}
 		});
 	t.is(resp.status, 200);
 });
```

Comment:
I am not sure this test makes sense to me. We are saying that the case officer would say that the questionnaire is complete and yet there is some missing/incorrect information?

Reply:
this has been corrected